src/ui/projects/utils.py

```diff
@@ -0,0 +1,22 @@
+"""Utilities functions used in ui."""
+from pydantic import validate_arguments
+
+SQLITE_INTEGER_LIMIT = 2 ** 63 - 1
+
+
+@validate_arguments
+def validate_int(
```

Comment:
Wouldn't it be better if it returned an `Optional[int]`?

Reply:
This PR is based upon #72 and this is from there. It is all ready fixed in that PR.